apps/mobile/src/modules/Cashu/qr/ScanCode.tsx

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable react-hooks/exhaustive-deps */
 import {BarcodeScanningResult, CameraView, useCameraPermissions} from 'expo-camera';
-import React, {useState} from 'react';
-import {Clipboard, Modal, Text, TouchableOpacity, View} from 'react-native';
+import jsQR from 'jsqr';
```

Comment:
This lib looks not added in the package.json. 
Why did you choose this one package?

Reply:
Oh my bad, I didn’t commit the package.json… will do.

and regarding the chosen library I needed one that worked on web, the expo one does not work on browsers